tsl/src/compression/create.c

```diff
@@ -583,13 +584,20 @@ create_compress_chunk_table(Hypertable *compress_ht, Chunk *src_chunk)
 	compress_chunk->hypertable_relid = compress_ht->main_table_relid;
 	compress_chunk->constraints = ts_chunk_constraints_alloc(1, CurrentMemoryContext);
 	namestrcpy(&compress_chunk->fd.schema_name, INTERNAL_SCHEMA_NAME);
-	snprintf(compress_chunk->fd.table_name.data,
-			 NAMEDATALEN,
-			 "compress%s_%d_chunk",
-			 NameStr(compress_ht->fd.associated_table_prefix),
-			 compress_chunk->fd.id);
 
-	;
+	/* Fail if we overflow the name limit */
+	namelen = snprintf(NameStr(compress_chunk->fd.table_name),
+					   NAMEDATALEN,
+					   "compress%s_%d_chunk",
+					   NameStr(compress_ht->fd.associated_table_prefix),
+					   compress_chunk->fd.id);
+
+	if (namelen >= NAMEDATALEN)
+		ereport(ERROR,
```

Comment:
```suggestion
	if (namelen >= NAMEDATALEN)
	{
		ereport(ERROR,
```

Reply:
I don't understand this suggestion. Why a single brace here?

Reply:
It was for adding the `pg_unreachable` below. Maybe could have been done better.

Reply:
Got it. FWIW, you can do multi-line suggestions in GitHub now.